SUMMARY.md

```diff
@@ -90,6 +72,14 @@
     - [Registers](./hw/ip/aes/doc/registers.md)
     - [Device Interface Functions](./sw/device/lib/dif/dif_aes.h)
     - [Checklist](./hw/ip/aes/doc/checklist.md)
+  - [Alert Handler](./hw/top_earlgrey/ip_autogen/alert_handler/README.md)
```

Comment:
Thanks for bringing this up, as this change is definitely a short-term hack; as you can see the 'Alert Handler' pages here (under the Generic HWIP block list) actually links to the ipgen-generated version under 'top_earlgrey' instead of to the 'ip_templates' version. I did this mostly because it does not make much sense to serve the Mako templates directly as documentation, they are ugly.

Thinking out-loud a bit here:
- Duplicating the same templated docs into each top_level's directory feels a bit ugly as a DNRY. (though at some point we would like to snapshot documentation of a top all-together as part of a release process).
- It would be nice to document the salient differences in _how_ we specialize/parameterize a generic HWIP for each top (register/memory maps etc) that makes it clear how a block is tailored for the needs of the top's use-case. It would be nice if this information could stand-alone from the documentation for the block as a whole.

There's probably other considerations here too. My first idea would be to put some work into ipgen to be a bit more flexible in how we generate documentation from the templates. WDYT?

Reply:
We will probably need to generate per-top documentation sets anyway. Perhaps in the unified site we should have a section for top-specific IPs, where for each one we create a page documenting what is parameterized, and then link to each top-specific full doc.

Something like this under a section for top-specific IPs.
- alert_handler
  - Parameterization
  - top1 alert_handler
  - top2 alert_handler
  - ...